### PR TITLE
ASP.NET Core 7 breaking change for SignalR Hub DI

### PR DESCRIPTION
**Pull request by serpent5**
Documents https://github.com/aspnet/Announcements/issues/479.

[Internal Preview](https://review.docs.microsoft.com/en-us/dotnet/core/compatibility/7.0?branch=pr-en-us-28473).

### CODE REVIEW

File: docs/core/compatibility/aspnet-core/7.0/signalr-hub-method-parameters-di.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+}
+```
+
+## Type of breaking change
+
+This change affects [source compatibility](../../categories.md#source-compatibility).
+
+## Reason for change
+
+We believe the likelihood of breaking apps to be very low as it isn't common to have a type in DI and as an argument in your Hub method at the same time.
```
Comment (gewarren) [inline docs/core/compatibility/aspnet-core/7.0/signalr-hub-method-parameters-di.md:51]: This isn't really a "reason for change"; nevertheless, I left a small suggestion.

```suggestion
The likelihood of breaking apps is low as it isn't common to have a type in DI and as an argument in your Hub method at the same time.
```

**Review by gewarren — APPROVED** [review]

File: docs/core/compatibility/aspnet-core/7.0/signalr-hub-method-parameters-di.md (hunk 1 above)
Comment (BrennanConroy) [reply docs/core/compatibility/aspnet-core/7.0/signalr-hub-method-parameters-di.md:51]: Isn't the "reason for change" in the context of making a breaking change? So this seems perfectly reasonable as the reason we were fine making a breaking change.

Comment (gewarren) [reply docs/core/compatibility/aspnet-core/7.0/signalr-hub-method-parameters-di.md:51]: That's not how I've seen this field used in the .NET breaking changes. I think of it more like "why it improves the experience". For example, [this issue](https://github.com/dotnet/docs/issues/28372).

Comment (serpent5) [reply docs/core/compatibility/aspnet-core/7.0/signalr-hub-method-parameters-di.md:51]: What should I do with this one? A similar announcement for DI in MVC's API controllers has this:

> This same behavior is already implemented in minimal APIs.
>
> The likelihood of breaking apps is low as it isn't common to have a type in DI and as an argument in your API controller action at the same time.

I see that as covering both why the change was made and why the change was acceptable as being breaking. Would you like to add a line similar to the first one in the example I gave above?

**Pull request merged**